### PR TITLE
Adding an initial button for profile creation

### PR DESCRIPTION
**Pull request by lialina**
This branch adds a Header and an initial Button components for profile creation

### CODE REVIEW

File: src/UI/Header.module.css [hunk 1]
```diff
@@ -0,0 +1,7 @@
+.header {
```
Comment (AObzh) [inline src/UI/Header.module.css:1]: Supposed to use styled-component for all styling stuff.

Comment (lialina) [reply src/UI/Header.module.css]: Updated to use styled-components.

File: src/UI/Header.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+const StyledHeader = styled.header`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  padding-top: 10px;
+  padding-bottom: 10px;
+`;
+
+function Header() {
```
[the remark this answers is not in the document]
Comment (lialina) [reply src/UI/Header.js]: Function declarations in components have been changed to arrow functions

File: src/UI/Header.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import Button from "../components/Button";
+import styled from "styled-components";
+
+const StyledHeader = styled.header`
```
[the remark this answers is not in the document]
Comment (lialina) [reply src/UI/Header.js]: Styled components have been moved to separate files.

**Review by AObzh — APPROVED** [review]

**Pull request merged**